xl_auth/templates/users/profile.html

```diff
@@ -31,7 +31,9 @@
 
     <div class="panel panel-default">
         <div class="panel-heading">
-            <h3>{{ _('Permissions') }}</h3>
+            <h3>{{ _('Permissions (Active Collections Only)') }}</h3>
+            <p>{{ _('Note: <em>Cataloguing Admin</em> is a new privilege that, in the tear future, \
```

Comment:
s/tear/near/

Reply:
Jäkla stavfel.. Alltid smyger de sig in någonstans!